src/main/java/org/osc/sdk/controller/api/SdnRedirectionApi.java

```diff
@@ -72,9 +86,12 @@
     List<NetworkElement> getNetworkElements(NetworkElement element) throws Exception;
 
     /**
+     * <p>
      * Creates an inspection hook on SDN controller, see {@link InspectionHookElement}
+     * In case of SFC, {@link InspectionPortElement#getElementId()} represents port chain id and the return value is the same port chain id
```

Comment:
How is this @link relevant to this method? And again i dont think you need to start the javadoc with <p>